src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -391,13 +408,17 @@ def iot_hub_create(cmd, client, hub_name, resource_group_name, location=None,
                    fileupload_sas_ttl=1,
                    fileupload_storage_authentication_type=None,
                    fileupload_storage_container_uri=None,
+                   fileupload_storage_identity=None,
                    min_tls_version=None,
-                   tags=None):
+                   tags=None,
+                   identities=None,
+                   identity_role=None,
+                   identity_scopes=None):
     from datetime import timedelta
     cli_ctx = cmd.cli_ctx
     if enable_fileupload_notifications:
         if not fileupload_storage_connectionstring or not fileupload_storage_container_name:
-            raise CLIError('Please specify storage endpoint(storage connection string and storage container name).')
+            raise CLIError('Please specify storage endpoint (storage connection string and storage container name).')
```

Comment:
Please use the specific error type `RequiredArgumentMissingError` instead of CLIError.